src/lib_gui/qt/project_wizard/content/QtProjectWizardContentPreferences.cpp

```diff
@@ -238,6 +238,15 @@ void QtProjectWizardContentPreferences::populate(QGridLayout* layout, int& row)
 		layout,
 		row);
 
+    m_logPath = new QtLocationPicker(this);
+    m_logPath->setPickDirectory(true);
+    addLabelAndWidget(QStringLiteral("Log Directory Path "), m_logPath, layout, row);
+    addHelpButton(
+        QStringLiteral("Log Directory Path"),
+        QStringLiteral("<p>Log file will be saved to this path.</p>"),
+        layout,
+        row);
+
```

Comment:
Our layouting here is still a little bit cumbersome. You need to call `row++` here, because unlike many other convenience methods, `addLabelAndWidget()` noes not do that on its own. Otherwise the location picker you added moves into the gap that follows.